repos/system_upgrade/el7toel8/actors/detectgrubconfigerror/tests/files/grub.correct_trailing_space

```diff
@@ -1,3 +1,3 @@
 GRUB_TERMINAL_OUTPUT="console"
-GRUB_CMDLINE_LINUX="console=tty0 crashkernel=auto console=ttyS0,115200n8 no_timer_check net.ifnames=0" 
-GRUB_DISABLE_RECOVERY="true"
\ No newline at end of file
+GRUB_CMDLINE_LINUX="console=tty0 crashkernel=auto console=ttyS0,115200n8 no_timer_check net.ifnames=0"
```

Comment:
regarding the name of the file and the fact this is possibly a real example, it should not be modified by this PR. discuss it with @jmikovic and @Rezney  - If the file reflect the real word, it really should not be touched in any way.

Reply:
Added this to ignore, the file suffix speaks on its own. Sorry for missing this

Reply:
This is based on a real world problem, I remember that this has been an issue in the past which had to be solved.